modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -46,27 +50,41 @@ for more information on manually defining a manifest file.
 
 . *Deploy the Operator.*
 
-.. Deploy an Operator by applying the Operator’s manifest to the desired namespace
-in the cluster:
+.. Create an OperatorGroup that specifies the namespaces that the Operator will
+be targeting. Create the following OperatorGroup in the namespace where you will
+create the CSV. In this example, the `default` namespace is used:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: memcached-operator-group
+  namespace: default
+  spec:
+    targetNamespaces:
+    - default
+----
+
+.. Apply the Operator's CSV manifest to the desired namespace in the cluster:
 +
 ----
 $ curl -Lo memcachedoperator.0.0.1.csv.yaml https://raw.githubusercontent.com/operator-framework/getting-started/master/memcachedoperator.0.0.1.csv.yaml
 $ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
 $ kubectl get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.status'
 ----
 
-.. After applying this manifest, nothing has happened yet, because the cluster does
-not meet the requirements specified in our manifest. Create the RBAC rules and
-`CustomResourceDefinition` for the Memcached type managed by the Operator:
+.. After applying this manifest, nothing has happened yet, because the cluster
+does not meet the requirements specified in our manifest. Create the RBAC rules
+and `CustomResourceDefinition` for the Memcached type managed by the Operator:
```

Comment:
@hasbro17 I was also wondering whether this sentence needed to change, now that the resources below have changed.

Reply:
@bergerhoffer The resources are still the same i.e the RBAC rules(Role, RoleBinding) and CRD. It's just the manifest paths that have changed.

But perhaps we can update this to more specifically list the exact RBAC resources that we're creating in the steps below, e.g:
```Markdown
Create the `Role`,`RoleBinding` and `ServiceAccount` to grant resource permissions to the operator, and the `CustomResourceDefinition` to define the `Memcached` type managed by the operator.
```